website/index.html

```diff
@@ -795,6 +815,7 @@ <h3>FAQ:</h3>
             <select id="main_hand_dd" class="weapon-select">
                 <option value="empty_weapon" selected></option>
                 <optgroup label="Swords">
+                    <option value="akil_zons_talonblade"> Akil'Zon's Talonblade</option>
```

Comment:
```suggestion
                    <option value="akilzons_talonblade"> Akil'Zon's Talonblade</option>
```
Following the items name convention.